test/bigtest/tests/sets/view-test.js

```diff
@@ -145,5 +152,86 @@ describe('Sets', () => {
         });
       });
     });
+
+    describe('View', () => {
+      describe('Expand/collapse all button', () => {
+        it('should be correct headline', () => {
```

Comment:
I see that [here](https://github.com/folio-org/ui-oai-pmh/pull/47/files#diff-de89be68f4805044d8dd38617cbda4c2R195) the "display" wording is used. I think it is more suitable and would suggest making it consistent in this file.
```suggestion
        it('should display correct headline', () => {
```

Reply:
Will be fixed in separate commit